### PR TITLE
Popup dialog

### PR DESCRIPTION
**Pull request by didierganthier**
### Popup Dialog
In this pull request we completed the following tasks:
- [x] Add structure and images to `index.html`
- [x] Add styles for the popup
- [x] Add Javascript interactions to show the popup
- [x] Add dynamic values for work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +42,82 @@ <h1 class="headline">I'm Didier<br/>Glad you're here!</h1>
...
+                <a href="#" class="work-link">
+                    See Source
+                    <img src="images/github.png" width="20" height="20" alt="arrow">
+                </a>
+            </div>
+        </div>
+    </div>
     <section  class="works-mobile" id="portfolio-mobile">
         <div class="work-container">
             <div class="work">
```
Comment (phelian23) [inline index.html]: - [ ] As requested in the project requirements, The projects section should be created dynamically using Javascript. all already created HTML code for the projects section list should be removed, and instead, it should be added via JavaScript. You should only have a container for the section in the HTML file.

File: style.css [hunk 2]
```diff
@@ -23,6 +43,92 @@
...
+
+  .work-subtitle-popup {
+    display: flex;
+    gap: 12px;
+    padding-left: 16px;
+    align-items: center;
+  }
+
+  .work-title-popup {
+    font-weight: 400;
```
Comment (phelian23) [inline style.css:86]: - [ ] Kindly make sure that your design matches that of the template. the description, buttons, and technologies are not visible on the desktop version.
Your design
Capture
Figma template
figma

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @didierganthier,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Following GitHub Flow
✔️ Following requirements
✔️ Descriptive PR Title
✔️ Descriptive PR Summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @didierganthier 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the requirements by the previous reviewer ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**